### DISCUSSION TITLE
Access database in Panel Provider

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#14766] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

i want to fetch brand logos from the db so how can i do that ?

**Comment by leandrocfe** [comment, marked as answer]
Use a `closure`

```php
$panel
->brandLogo(fn (): string => YourModel::...))
```

**Reply by zeeshantariq08** [reply]
thanks , it worked !
